test/channels/retro_channel_test.exs

```diff
@@ -139,7 +139,7 @@ defmodule RemoteRetro.RetroChannelTest do
 
 
     @tag user: @mock_user
-    @tag idea: %Idea{category: "sad", body: "doggone keeper", author: "Maryanne"}
```

Comment:
"So long, Maryanne! It's time that we began to laugh and cry and cry and laugh about it all again!"